alerts/machine-config-operator/ExtremelyHighIndividualControlPlaneMemory.md

```diff
@@ -0,0 +1,27 @@
+# ExtremelyHighIndividualControlPlaneMemory
+
+## Meaning
+
+This alert fires when the memory usage per an instance within control plane
+nodes exceeds 90% of total memory available per a node.  
+This is a critical level alert.
+
+## Impact
+
+At such very low free memory, the cluster becomes unstable and expected slow
+responses from kube-apiserver or failing requests specially on etcd.
+Moreover, OOM kill is expected which negatively influences the pod scheduling.
+
+## Diagnosis
+
+To fix this, increase memory of the affected node of control plane nodes.
+
+## Mitigation
+
+Using `HighOverallControlPlaneMemory` alert you can mitigate this situation.
+`HighOverallControlPlaneMemory` is a warning level alert, which fires
+when the total memory usage across all
+control-plane nodes crosses 60% for more than 1 hour.
+
+You may want to scale up the memory of the control plane node if this situation
+stayed up for long time and before the critical alert fires up.
```

Comment:
s/may want to/could
s/stayed/stays
s/and before the critical alert fires up/to prevent the critical alert from firing (Is this correct?)

Reply:
+1